Kinvey.Core/Store/MultiInsertRequest.cs

```diff
@@ -300,5 +340,43 @@ private async Task<KinveyMultiInsertResponse<T>> HandleNetworkRequestAsync(IList
 
             return kinveyDataStoreResponse;
         }
+
+        private async Task<KinveyMultiInsertResponse<T>> HandleMultiInsertRequestAsync(IList<T> entities)
+        {
+            var response = new KinveyMultiInsertResponse<T>();
+
+            try
+            {
+                var multiInsertRequest = Client.NetworkFactory.BuildMultiInsertRequest<T, KinveyMultiInsertResponse<T>>(Collection, entities.ToList());
+                response = await multiInsertRequest.ExecuteAsync();
+            }
+            catch (KinveyException exeption)
```

Comment:
`exception` missing `c`

Reply:
Thank you @vishwakrishnamurthy ! I will correct it.